adafruit_io/adafruit_io.py

```diff
@@ -195,22 +190,39 @@ def _on_message_mqtt(self, client, topic, payload):
 
     # pylint: disable=not-callable
     def _on_subscribe_mqtt(self, client, user_data, topic, qos):
-        """Runs when the client calls on_subscribe.
-        """
+        """Runs when the client calls on_subscribe."""
         if self._logger:
             self._client._logger.debug("Client called on_subscribe")
         if self.on_subscribe is not None:
             self.on_subscribe(self, user_data, topic, qos)
 
     # pylint: disable=not-callable
     def _on_unsubscribe_mqtt(self, client, user_data, topic, pid):
-        """Runs when the client calls on_unsubscribe.
-        """
+        """Runs when the client calls on_unsubscribe."""
         if self._logger:
             self._client._logger.debug("Client called on_unsubscribe")
         if self.on_unsubscribe is not None:
             self.on_unsubscribe(self, user_data, topic, pid)
 
+    def add_feed_callback(self, feed_key, callback_method):
+        """Executes callback_method whenever a message is
+        received on feed_key.
+        :param str feed_key: Adafruit IO feed key.
+        :param str callback_method: Name of callback method.
+
+        """
+        self._client.add_topic_callback(
+            "{0}/feeds/{1}".format(self._user, feed_key), callback_method
+        )
+
+    def remove_feed_callback(self, feed_key):
+        """Removes a previously registered callback method
+        from executing whenever feed_key receives new data.
+        :param str feed_key: Adafruit IO feed key.
+
+        """
+        self._client.remove_topic_callback("{0}/feeds/{1}".format(self._user, feed_key))
+
     def loop(self):
         """Manually process messages from Adafruit IO.
         Call this method to check incoming subscription messages.
```

Comment:
Please add a sentence here about how it will call the event handlers depending on the type of message.

Reply:
addressing this here: https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/pull/47